plugins/shared/audit.js

```diff
@@ -27,9 +29,17 @@ function createWhitelist(ruleName) {
 // Audits for a single rule (by name) and returns the results for only that
 // rule
 function audit(ruleName) {
+    // Monkey-patch `matchSelector` for our jsdom testing environment,
+    // using jQuery for optimal browser support.
+    //
+    // https://github.com/GoogleChrome/accessibility-developer-tools/pull/189
+    $.axs.browserUtils.matchSelector = (node, selectorText) => {
```

Comment:
This is copied from LinkTextPlugin, and I unfortunately couldn't find a good way to prevent duplicating code.

It's very temporary, you'll see my PR has already been merged upstream, and a new release of ADT will be pushed to NPM soon.